resources/config/config.yaml

```diff
@@ -38,6 +43,8 @@ threads:
   spades: 8
   megahit: 8
   metaphlan: 4
+  bowtie2_build: 8
```

Comment:
Maybe `host_filter` here should be changed to simply `bowtie2`, since it not specifies how many threads to use on two different rules.